packages/opentelemetry-instrumentation-milvus/opentelemetry/instrumentation/milvus/wrapper.py

```diff
@@ -92,91 +93,91 @@ def count_or_none(obj):
 @dont_throw
 def _set_insert_attributes(span, kwargs):
     _set_span_attribute(
-        span, "db.milvus.insert.collection_name", kwargs.get("collection_name")
+        span, f"{AISpanAttributes.MILVUS_INSERT}.collection_name", kwargs.get("collection_name")
```

Comment:
This also needs to be a constant. So there should be something in `SpanAttributes` that's called
`MILVUS_INSERT_COLLECTION_NAME` that is used here. The reason we need it is because there are observability system that automatically generates dashboards etc. from these constants so we need to be explicit about them.